src/stream/src/executor/monitor/streaming_stats.rs

```diff
@@ -287,6 +288,13 @@ impl StreamingMetrics {
                 registry
             )
             .unwrap();
+        let actor_output_buffer_blocking_duration_ns =
```

Comment:
Can this really work? I am thinking that

- The `LabelGuarded` metrics were implemented by [overriding its `drop()` ](https://github.com/risingwavelabs/risingwave/blob/da28570ab58385ca5850f659ed0f96e3d9843151/src/common/metrics/src/guarded_metrics.rs#L327) to remove the specific `labels` from Prometheus client.
- Here, multiple actors share the same specific `labels`

Thus, when a metrics being dropped such as `drop MV`, I guess it will panic at `.expect("should exist")`.

Reply:
I think this was addressed by https://github.com/risingwavelabs/risingwave/pull/12882, where `Guarded` with the same labels share the same underlying ref count, thus can be created individually.